src/durableClient/DurableClient.ts

```diff
@@ -492,8 +493,24 @@ export class DurableClient implements types.DurableClient {
                 .replace(this.instanceIdPlaceholder, instanceIdPath);
         }
 
+        // Get the current active span
+        const currentSpan = OpenTelemetryApi.trace.getSpan(OpenTelemetryApi.context.active());
+        let traceParent = "";
+        let traceState = "";
+        const headers: Record<string, string> = {};
+        if (currentSpan) {
+            const spanContext = currentSpan.spanContext();
+            traceParent = `00-${spanContext.traceId}-${
+                spanContext.spanId
+            }-${spanContext.traceFlags.toString(16).padStart(2, "0")}`;
+            traceState = spanContext.traceState ? spanContext.traceState.serialize() : "";
+
+            headers["traceparent"] = traceParent;
+            headers["tracestate"] = traceState;
```

Comment:
I think there's a discrepency between the PR description and the implementation. In the PR description, you said that you couldn't use `traceparent` and `tracestate` because those values were being overwritten. How was this resolved?

Reply:
I thought I had updated that, but I must not have saved the change. Thanks for pointing this out. I just updated the description.